src/platform/telink/BUILD.gn

```diff
@@ -32,6 +32,8 @@ static_library("telink") {
     "../Zephyr/SystemTimeSupport.cpp",
     "../Zephyr/ZephyrConfig.cpp",
     "../Zephyr/ZephyrConfig.h",
+    "BLEAdvertisingArbiter.cpp",
```

Comment:
Maybe you could include those from `../Zephyr` instead of duplicating? Or you needed to make any changes to that?

Reply:
Fixed